### DISCUSSION TITLE
ZIM-template Ecodiv-responsive

### DISCUSSION THREAD

**Discussion by aloboa** [zim-desktop-wiki/zim-desktop-wiki#2222] [Q&A]
I export to html using the Ecodiv-Responsive  (https://github.com/ecodiv/ZIM-templates/tree/master/Ecodiv-responsive) as template, but what I get is quite different to the example http://vegetationmap4africa.org/
Why?

**Comment by introt** [comment]
Feel free to ask the template maintainer over at https://github.com/ecodiv/ZIM-templates/issues - their third issue seems to imply there's some manual magic involved
